subprojects/docs/src/docs/release/notes.md

```diff
@@ -138,6 +138,16 @@ The `CodeNarc` task also produces clickable links in failure messages to human-r
 
 The HTML report generated by `CodeNarc` has been updated to produce violation reports with sortable columns. 
 
+### Group opposite boolean task options together
+
+The console output of the `help` task renders options in alphabetical order.
+
+For better readability, this has been changed for opposite boolean task options.
```

Comment:
```suggestion
For better readability, the ordering has been changed to task options followed immediately by their opposite boolean task options.
```

Reply:
I'm not sure I understand the reworded text. The ordering only changes for opposite boolean task options, they are only a subset of the rendered options and all other options are still sorted alphabetically.

Reply:
I agree that with the other changes this suggestion now feels weird. Let's keep it like that it is more correct.